taxi/views.py

```diff
@@ -13,3 +15,31 @@ def index(request):
     }
 
     return render(request, "taxi/index.html", context=context)
+
+
+class ManufacturerListView(generic.ListView):
+    model = Manufacturer
+    queryset = Manufacturer.objects.all().order_by("name")
+    paginate_by = 5
+
+
+class CarListView(generic.ListView):
+    model = Car
+    paginate_by = 5
+    queryset = (Car.objects.select_related("manufacturer").
+                all().order_by("model"))
+
+
+class CarDetailView(generic.DetailView):
+    model = Car
+
+
+class DriverListView(generic.ListView):
+    model = Driver
+    paginate_by = 5
+
+
+class DriverDetailView(generic.DetailView):
+    model = Driver
+    queryset = (Driver.objects.prefetch_related("cars__manufacturer").
+                all().order_by("username"))
```

Comment:
```suggestion
    queryset = (Driver.objects.prefetch_related("cars__manufacturer").
               .order_by("username"))
```

`all()` method redundant here